pkgs/development/python-modules/pyTwitchAPI/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, aiohttp
+, python-dateutil
+, requests
+, websockets
+}:
+
+buildPythonPackage rec {
+  pname = "pyTwitchAPI";
+  version = "2.3.0";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "Teekeks";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-ax3FHyyyRfXSWKsoUi8ao5TL2alo0bQP+lWiDaPjf34=";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+    python-dateutil
+    requests
+    websockets
+  ];
+
+  doChecks = false;
+
+  pythonImportsCheck = [ "aiohttp" "dateutil" "requests" "twitchAPI" "websockets" ];
```

No need to check the import of the used modules.

```suggestion
  pythonImportsCheck = [ "twitchAPI" ];
```

---

Changes implemented